pulsetranslator/pulsetranslator.py

```diff
@@ -320,12 +320,24 @@ def on_pulse_message(self, data, message=None):
             # Lets fetch it via the release tag and the hg.m.o REST API
             if builddata['tree'].startswith('release-') and builddata['revision'] in [None, 'None']:
                 try:
-                    url = 'https://hg.mozilla.org/releases/{tree}/json-rev/{release_tag}'.format(
-                        tree=builddata['tree'].split('release-')[1],
-                        release_tag=builddata['release']
-                    )
+                    # Map for platforms which change their id
+                    platform_map = {
+                        'linux': 'linux-i686',
+                        'linux64': 'linux-x86_64',
+                        'macosx64': 'mac',
```

Comment:
I have added those but would leave the final decision to @jonallengriffin. As stated those platforms as listed here are only those which change their ids.